### ISSUE TITLE
Customizable default lane arrows and/or AI vehicles issue

### ISSUE THREAD

**Issue by Eduardo12X** [CitiesSkylinesMods/TMPE#1763]
For better understanding, I'll give an example that's the problem I'm having, by default, the second lane arrow (or the middle lane arrow) from six lane roads is set to the vehicles go to the front when there are 3 others 6 lane roads connected (you can check the screenshot if it's confusing), but the problem of this default lane arrows configuration is most of the vehicles will focus to go to the right or left of the road with it, consequently, the traffic can accumulate easily and will do the middle of the road most of the time useless ("because will be just decorative", what's bad to decrease the traffic) 
That's why it's a nice idea to let the default configuration of the lane arrows customizable because when you expand the city, the amount of the four roads connection with six lanes are (or will be) too big to change manually. Also, is interesting improve the vehicles AI (if it's the mod case and if is the right place to report this), since can happen even though I set the modified lane arrows below, the vehicles can still insist to go to the right or left instead in the middle of the road at least 33-50% of the times depending on what direction them want to go (Notes section can explain it in the clear way)

### Defaulf configuration in this case

![Screenshot 2024-02-12 011326](https://github.com/CitiesSkylinesMods/TMPE/assets/85649279/066fec28-ada9-40ac-9421-c29a964de7e5)

### Modified Lane Arrows

![Screenshot 2024-02-12 011451](https://github.com/CitiesSkylinesMods/TMPE/assets/85649279/e7e2921d-b689-420b-93e6-0e247ab76332)

### Notes or questions:

The modified lane arrows screenshot is the configuration I wanted to every case of those connections with four roads with six lanes be like that way. About the possible vehicles AI issue, by default, the vehicles can just ignore the middle's road exists, so the idea I have is given a 33-50% chance of the vehicle go to the middle road, since it's 33-34% if the vehicle just can go to the left, right and middle in case to have 3 paths, or 50% in case of the vehicle can just go to the left or right, and middle, I mean, 2 paths

**Comment by kvakvs** [comment]
We already have dynamic (random) lane change option and it will help you mix the vehicles and lanes better, if you set it to 50-75%.

**Comment by krzychu124** [comment]
Exactly, dynamic selection slider tells the system (more or less) what percentage of vehicles should try to check if there is better lane and depends on vehicle speed (slower = more checks) vehicle may even check it every node. If internally calculated traffic flow of lane is not much better than the one which is original path lane vehicle won't switch lane, similarly when much better lane is too far (would require multiple changes) vehicle won't try to switch. 

Moreover it also applies to vehicles following much slower vehicle - that situation may turn into fake overtaking behavior when speed difference is big enough (e.g.: vehicle behind could move 20% faster after switching lane, still following speed limit on the road)

**Comment by Eduardo12X** [comment]
Well, I tested the feature you said, looks like really efficient, so don't consider the AI vehicle feedback I did anymore, and just consider the customization of the default lane arrows, ok?
